pkg/operator/ceph/cluster/cluster.go

```diff
@@ -401,9 +401,10 @@ func diffImageSpecAndClusterRunningVersion(imageSpecVersion cephver.CephVersion,
 			clusterRunningVersion := *version
 
 			// If this is the same version
+			// We return 'true' so that we restart using the Ceph checks
```

Comment:
Don't we need to check if the image name changed? Or where is the check for the image name change?

Reply:
This happens way earlier here https://github.com/rook/rook/blob/master/pkg/operator/ceph/cluster/controller.go#L589

Reply:
I see, it's checked elsewhere. Should we just `return true` here? Looks like no need to check if `IsIdentical()`, `IsSuperior()`, or `IsInferior()`.